### ISSUE TITLE
active?

### ISSUE THREAD

**Issue by sherodtaylor** [variantjs/core#17]
Is this still actively being worked on?

**Comment by invokermain** [comment]
Would also love to know what the future plan is, its such a great concept for a component library (nothing else out there like it).

@alfonsobries seems very busy with other projects, so I imagine its not going to be actively being worked on in the near future?

But it is good to know roughly what the future plan is. I have written stuff that uses this library and have patiently been waiting for more components. However, if there is a plan to only say bugfix for the near future, then will look at other libraries to add in the missing components. So would be helpful to know to make that choice.

Would even be good to know that contributions would actively be accepted, would happily merge request some of the issues I raised in the Vue library. Currently, the issues are not replied to.

**Comment by alfonsobries** [comment]
@sherodtaylor @invokermain yes it is, I've been struggling with finding time since recently have a new child

It's very likely that is going to have slow development for a couple of weeks more though

**Comment by sherodtaylor** [comment]
Congrats! No worries I was just curious about the support because the vue repository seems to have more work.
